src/components/ui/InfiniteList/Styled.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import styled, { keyframes } from 'styled-components';
+
+import { baseStyles, baseSpacing } from '../Base';
+import { InfiniteListProps } from './';
+
+// Create the keyframes
+const rotate = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+
+export const StyledInfiniteList = styled.ul<InfiniteListProps>`
+  overflow-y: scroll;
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  padding: 0;
+
+  /* disable scrolling with fetching */
```

Comment:
while*

Reply:
Thanks. Fixed.

I don't know why, but I can't reply to your comment about `IntersectionObservers`. I looked into this briefly, but I was able to achieve my goal without using it. I'll look further into it if it'll be advantageous.